### PR TITLE
Setup rule

### PR DESCRIPTION
**Pull request by Thanghand**
Update model and set up rule

### CODE REVIEW

File: blockchain/models/org.bitship.cto [hunk 1]
```diff
@@ -4,104 +4,92 @@
...
+asset Warehouse identified by warehouseId {
+  o String warehouseId
+  o Package[] packages
+  o Location location
 }
 
-transaction ShipmentSession {
+/* Transactions */
+
+transaction TransferShippment {
```
Comment (tiengtinh) [inline blockchain/models/org.bitship.cto]: how about `ShipmentTransfer`
just like "BalanceTranfer"

File: blockchain/models/org.bitship.cto [hunk 2]
```diff
@@ -4,104 +4,92 @@
...
-  o Package[] packages
+  o Package packages
   o SessionStatus status
-
   o Location location
 }
 
-transaction WarehouseSession {
-  o Package[] packages
+transaction TransferWarhouse {
```
Comment (tiengtinh) [inline blockchain/models/org.bitship.cto:87]: similarly "WarehouseTransfer"

File: blockchain/models/org.bitship.cto [hunk 3]
```diff
@@ -4,104 +4,92 @@
...
 }
 
-transaction ShipmentSession {
+/* Transactions */
+
+transaction TransferShippment {
   --> ShipmentVehicle vehicle
   --> Shipper shipper
-  o Package[] packages
+  o Package packages
```
Comment (tiengtinh) [inline blockchain/models/org.bitship.cto]: `package` no `s`

File: blockchain/models/org.bitship.cto [hunk 4]
```diff
@@ -4,104 +4,92 @@
...
-/* Transactions */
-
-enum SessionStatus {
-  o IN_PROGRESS
-  o DONE
+asset ShipmentVehicle identified by shipmentVehicleId {
+  o String shipmentVehicleId
+  o Package[] packages
+  o Location location
+  --> Shipper owner
```
Comment (tiengtinh) [inline blockchain/models/org.bitship.cto]: `driver`

File: blockchain/models/org.bitship.cto [hunk 5]
```diff
@@ -4,104 +4,92 @@
...
-  o NOT_OK
-}
-
-enum InspectionType {
-  o CHECK_IN
-  o CHECK_OUT
-}
-
-transaction Inspection {
+transaction ReportInspector {
```
Comment (tiengtinh) [inline blockchain/models/org.bitship.cto:92]: `Inspector` is a person
should be `ReportInspection`

**Review by tiengtinh — CHANGES_REQUESTED** [review]

**Comment by Thanghand** [comment]
Thanks for your feedback

**Pull request closed without merging**